hr_maintenance_workspaces/models/workspace.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 SDi Digital Group (https://www.sdi.es/odoo-cloud)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Workspace(models.Model):
+    _name = 'maintenance.workspace'
+    _inherit = 'mail.thread'
+    _description = 'Workspace'
+
+    name = fields.Char(
+        string='Name',
+        required=True,
+    )
+    description = fields.Char(
+        string='Description',
+    )
+    location = fields.Char(
+        string='Location',
+    )
+    equipment_ids = fields.One2many(
+        comodel_name='maintenance.equipment',
+        inverse_name='workspace_id',
+    )
+    equipment_count = fields.Integer(
+        name='Equipments',
+        compute='_compute_equipment_count',
+    )
+    employee_ids = fields.Many2many(
+        comodel_name='hr.employee',
+        string='Employees',
+    )
+    employee_count = fields.Integer(
+        string='Employees',
+        compute='_compute_employee_count',
+    )
+    employee_equipment_count = fields.Integer(
+        string='Employee Equipment Count',
+        compute='_compute_employee_equipment_count',
+    )
+
+    @api.depends('equipment_ids')
+    def _compute_equipment_count(self):
+        for workspace in self:
+            workspace.equipment_count = len(workspace.equipment_ids)
+
+    @api.depends('employee_ids')
+    def _compute_employee_count(self):
+        for workspace in self:
+            workspace.employee_count = len(workspace.employee_ids)
+
+    @api.depends('employee_ids')
+    def _compute_employee_equipment_count(self):
+        for workspace in self:
+            workspace.employee_equipment_count = 0
+            if len(workspace.employee_ids) > 0:
+                for employee in workspace.employee_ids:
+                        workspace.employee_equipment_count += \
+                            len(employee.equipment_ids)
```

Comment:
This could potentially lead to multiple writes to the database. Instead create a temporary variable to hold the count and then assign it to `workspace.employee_equipment_count` after you exit the loop.